src/odemis/gui/cont/settings.py

```diff
@@ -212,11 +211,11 @@ def add_metadata(self, key, value):
             else:
                 # Still try to beautify a bit if it's a number
                 if (
-                    isinstance(value, (int, long, float)) or
```

Comment:
You shouldn't have these changes related to "long". They are already in the latest master. (But I guess git can handle them fine so not need to worry too much)